hooks/useControllerPricesData/useControllerPricesData.ts

```diff
@@ -138,87 +137,3 @@ export function useControllerPricesData(): ControllerPricesDataReturn {
     error: null,
   };
 }
-
-function targets(
-  targetUpdates: TargetUpdatesByControllerQuery['targetUpdates'],
-  targetUpdate: TargetUpdate,
-  underlyingDecimals: ethers.BigNumberish,
-) {
-  const sortedTargets = [...targetUpdates].sort(
-    (a, b) => a.timestamp - b.timestamp,
-  );
-
-  // get what target would be if updated at this moment and add to array
-  if (sortedTargets.length > 0) {
-    const { newTarget, timestamp } = targetUpdate;
-    sortedTargets.push({
-      id: 'filler',
-      newTarget,
-      timestamp,
-    });
-  }
-
-  const formattedTargets = sortedTargets.map((target) => {
-    const t = target.timestamp;
-    return {
-      value: parseFloat(
-        ethers.utils.formatUnits(
-          ethers.BigNumber.from(target.newTarget),
-          underlyingDecimals,
-        ),
-      ),
-      time: t as UTCTimestamp,
-    };
-  });
-  return formattedTargets;
-}
-
-/**
- * On one swap there was a tremendous market price which ruins the chart.
- * Let's filter that one out.
- */
-const OUTLIER_TIMESTAMP = 1675965383;
-
-function marks(
-  swaps: SwapsByPoolQuery['swaps'],
-  baseCurrency: Token,
-  quoteCurrency: Token,
-  token0: Token,
-) {
-  const now = Math.floor(Date.now() / 1000);
-
-  const sortedSwaps = [...swaps].sort(
```

Comment:
not related, but seems we should be able to sort in the swap query itself? 🤔

Reply:
good point